ecs-agent/third_party.tpl

```diff
@@ -0,0 +1,7 @@
+{{ range . -}}
```

Comment:
do we need to commit this file to code?   I used https://github.com/google/go-licenses to pull these for `/agent` and didn't commit this file there.

Reply:
Update - talked offline and we'll move this into a make target workflow.  approved.